sources/engine/Xenko/Rendering/ParameterCollection.cs

```diff
@@ -175,18 +175,6 @@ private unsafe Accessor GetValueAccessorHelper(ParameterKey parameterKey, int el
             return new Accessor(memberOffset, elementCount);
         }
 
-        /// <summary>
-        /// Gets pointer to directly copy blittable values.
-        /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="parameter"></param>
-        /// <returns></returns>
-        public unsafe IntPtr GetValuePointer<T>(ValueParameter<T> parameter) where T : struct
-        {
-            fixed (byte* dataValues = DataValues)
-                return (IntPtr)dataValues + parameter.Offset;
```

Comment:
Good catch! I think it was a remnant of the time when `DataValues` was either native or pinned.